File: Acceso_a_datos/Manejo_de_Ficheros/src/AccesoDirecto/Persona.java
package AccesoDirecto;

public class Persona {
	char[] nombre = new char[10];
	int edad;
	public Persona(char[] nombre, int edad) {
		super();
		this.nombre = nombre;
		this.edad = edad;
	}
	public char[] getNombre() {
		return nombre;
	}
	public void setNombre(char[] nombre) {
		this.nombre = nombre;
	}
	public int getEdad() {
		return edad;
	}
	public void setEdad(int edad) {
		this.edad = edad;
	}
	
}

File: Programacion_multimedia/CodeKotlin1/app/src/main/java/Persona.kt
class Persona (nombre:String, apellido:String, edad:Int) {
    var nombre = nombre
    var apellido = apellido
    var edad = edad
}

fun main(){
    val persona1 = Persona("Pepito", "Grillo", 7)
    print(persona1.nombre)
    print(persona1.apellido)
    print(persona1.edad)
}
